### PR TITLE
feat/thing-group-api

### PR DESCRIPTION
**Pull request by Kerwin-Jhong**
Fixes #

### CODE REVIEW

File: lambda-assets/thing-groups/get-thing-group/app.ts [hunk 1]
```diff
@@ -11,10 +11,12 @@ export async function handler(event: { [key: string]: any }) {
         thingGroupName: request.parameter('thingGroupName'),
       }),
     );
-    return response.json({
-      thingGroup,
-    });
+    return response.json(thingGroup);
```
Comment (MinCheTsai) [inline lambda-assets/thing-groups/get-thing-group/app.ts:14]: {
  thingGroup: {
     name
  }
}

File: lambda-assets/thing-groups/list-thing-groups/app.ts [hunk 2]
```diff
@@ -11,10 +11,8 @@ export async function handler(event: { [key: string]: any }) {
         nextToken: request.get('nextToken', undefined),
       }),
     );
-    return response.json({
-      thingGroups,
-    });
+    return response.json(thingGroups);
```
Comment (MinCheTsai) [inline lambda-assets/thing-groups/list-thing-groups/app.ts:14]: {
  thingGroups,
  nextToken,
}

Comment (Kerwin-Jhong) [reply lambda-assets/thing-groups/list-thing-groups/app.ts:14]: 已確認output格式

File: test/lambda/thing-group.test.ts [hunk 3]
```diff
@@ -1,163 +1,454 @@
...
+    thingGroupName: expected.thingGroupName,
+  }).resolves({});
+  const response = await deleteThingGroup.handler({
+    pathParameters: {
+      thingGroupName: expected.thingGroupName,
+    },
+  });
+  const body = JSON.parse(response.body);
+  expect(response.statusCode).toEqual(200);
+  expect(body.deleted).toEqual(true);
```
Comment (MinCheTsai) [inline test/lambda/thing-group.test.ts:278]: restore

**Review by MinCheTsai — APPROVED** [review]

**Pull request merged**